src/_swift-demangle

```diff
@@ -1,12 +1,14 @@
 #compdef swift-demangle
 
 _arguments \
-  '*:Symbol(s):->' \
+  '*:Symbol(s): ' \
```

Comment:
The change to the empty space is according to `man zshcompsys`:

               (single unquoted space)
                     This  is  useful  where an argument is required but it is
                     not possible or desirable to  generate  matches  for  it.
                     The  message will be displayed but no completions listed.
                     Note that even in this case the colon at the end  of  the
                     message  is needed; it may only be omitted when neither a
                     message nor an action is given.